pyproject.toml

```diff
@@ -37,6 +37,7 @@ aiosqlite = "^0.17.0"
 
 # Development dependencies
 # ========================
+pyhumps = "^3.0.2"
```

Comment:
Put this above the title.

Reply:
I try to use poetry to add and remove things instead of manually editing the `pyproject.toml` file.  So I don't really think about updating it for documentation purposes.